[BUG] - Error during login. Deserialization failed

### Description

When trying to login to my Jellyfin 10.11.1 server with my user/password credentials or Quickconnect I get the error:
Error during login

Deserialization failed

Cause: Field 'overview' is required for type with serial name

'org.jellyfin.sdk.model.api.PackageInfo', but it was missing at path:

$.CompletedInstallations [1].PackageInfo

With version 0.1.2 it was working fine but even this version throws the same error now.

### Reproduction steps

1. Username: xxx
2. Password: xxx
3. Click on: Login
4. See error

### App Version

v0.2.4

### Server Version

10.11.1

### Device

NVIDIA Shield 2017

### Logs

```bash
11-02 11:13:43.509 16550 16572 I org.jellyfin.sdk.api.okhttp.OkHttpClient: GET http://192.168.0.209:8096/Branding/Configuration
11-02 11:13:43.534 16550 16572 I org.jellyfin.sdk.api.okhttp.OkHttpClient: GET http://192.168.0.209:8096/System/Info/Public
11-02 11:13:43.541 16550 16572 I org.jellyfin.sdk.api.okhttp.OkHttpClient: GET http://192.168.0.209:8096/Users/Me
11-02 11:13:43.554 16550 16572 I org.jellyfin.sdk.api.okhttp.OkHttpClient: GET http://192.168.0.209:8096/DisplayPreferences/livetv?client=jellyfin-androidtv
11-02 11:13:43.582 16550 16572 I org.jellyfin.sdk.api.okhttp.OkHttpClient: GET http://192.168.0.209:8096/Shows/NextUp?limit=10&fields=CanDelete&fields=ChannelInfo&fields=Chapters&fields=ChildCount&fields=CumulativeRunTimeTicks&fields=DateCreated&fields=DisplayPreferencesId&fields=Genres&fields=ItemCounts&fields=MediaSourceCount&fields=MediaSources&fields=MediaStreams&fields=Overview&fields=Path&fields=PrimaryImageAspectRatio&fields=Taglines&fields=Trickplay&imageTypeLimit=1&enableTotalRecordCount=true&disableFirstEpisode=false&enableResumable=false&enableRewatching=false
11-02 11:13:43.582 16550 16573 I org.jellyfin.sdk.api.okhttp.OkHttpClient: GET http://192.168.0.209:8096/UserItems/Resume?limit=10&fields=CanDelete&fields=ChannelInfo&fields=Chapters&fields=ChildCount&fields=CumulativeRunTimeTicks&fields=DateCreated&fields=DisplayPreferencesId&fields=Genres&fields=ItemCounts&fields=MediaSourceCount&fields=MediaSources&fields=MediaStreams&fields=Overview&fields=Path&fields=PrimaryImageAspectRatio&fields=Taglines&fields=Trickplay&mediaTypes=Video&imageTypeLimit=1&includeItemTypes=Episode&includeItemTypes=Movie&enableTotalRecordCount=true&enableImages=true&excludeActiveSessions=true
11-02 11:13:43.614 16550 16573 I org.jellyfin.sdk.api.okhttp.OkHttpClient: GET http://192.168.0.209:8096/DisplayPreferences/usersettings?client=emby
11-02 11:13:44.144 16550 16587 I org.jellyfin.sdk.api.okhttp.OkHttpClient: GET http://192.168.0.209:8096/Items/Latest?fields=CanDelete&fields=ChannelInfo&fields=Chapters&fields=ChildCount&fields=CumulativeRunTimeTicks&fields=DateCreated&fields=DisplayPreferencesId&fields=Genres&fields=ItemCounts&fields=MediaSourceCount&fields=MediaSources&fields=MediaStreams&fields=Overview&fields=Path&fields=PrimaryImageAspectRatio&fields=Taglines&fields=Trickplay&includeItemTypes=Movie&isPlayed=false&limit=50&groupItems=true
11-02 11:13:44.145 16550 16574 I org.jellyfin.sdk.api.okhttp.OkHttpClient: GET http://192.168.0.209:8096/Items/Latest?fields=CanDelete&fields=ChannelInfo&fields=Chapters&fields=ChildCount&fields=CumulativeRunTimeTicks&fields=DateCreated&fields=DisplayPreferencesId&fields=Genres&fields=ItemCounts&fields=MediaSourceCount&fields=MediaSources&fields=MediaStreams&fields=Overview&fields=Path&fields=PrimaryImageAspectRatio&fields=Taglines&fields=Trickplay&includeItemTypes=Movie&includeItemTypes=Series&isPlayed=false&limit=50&groupItems=true
11-02 11:13:44.145 16550 16572 I org.jellyfin.sdk.api.okhttp.OkHttpClient: GET http://192.168.0.209:8096/Items/Latest?fields=CanDelete&fields=ChannelInfo&fields=Chapters&fields=ChildCount&fields=CumulativeRunTimeTicks&fields=DateCreated&fields=DisplayPreferencesId&fields=Genres&fields=ItemCounts&fields=MediaSourceCount&fields=MediaSources&fields=MediaStreams&fields=Overview&fields=Path&fields=PrimaryImageAspectRatio&fields=Taglines&fields=Trickplay&includeItemTypes=Episode&isPlayed=false&limit=50&groupItems=true
11-02 11:13:45.669 16550 16573 I org.jellyfin.sdk.api.okhttp.OkHttpClient: GET http://192.168.0.209:8096/UserViews?includeHidden=false
11-02 21:19:31.509  3690  4657 I ActivityTaskManager: START u0 {act=android.intent.action.MAIN cat=[android.intent.category.LEANBACK_LAUNCHER] flg=0x10000000 pkg=com.github.damontecres.wholphin cmp=com.github.damontecres.wholphin/.MainActivity} from uid 10067
11-02 21:19:31.559  3690  3717 I ActivityManager: Start proc 28486:com.github.damontecres.wholphin/u0a103 for pre-top-activity {com.github.damontecres.wholphin/com.github.damontecres.wholphin.MainActivity}
11-02 21:20:29.052  3690  6749 W NotificationService: Toast already killed. pkg=com.github.damontecres.wholphin token=android.os.BinderProxy@23effce
11-02 21:47:17.142  3690  3797 D InputDispatcher: Waiting to send key to Window{b7cd389 u0 com.github.damontecres.wholphin/com.github.damontecres.wholphin.MainActivity} because there are unprocessed events that may cause focus to change
11-02 21:47:19.038 28486 28517 I OkHttpClient: POST http://192.168.0.209:8096/QuickConnect/Initiate
11-02 21:47:20.775  3690  5434 W NotificationService: Toast already killed. pkg=com.github.damontecres.wholphin token=android.os.BinderProxy@d465d00
11-02 21:47:22.193  3690  3798 D WindowManager: isOnHomeScreen mLastTopComponent: null, componentName: ComponentInfo{com.github.damontecres.wholphin/com.github.damontecres.wholphin.MainActivity}
11-02 21:47:22.302  3690  3798 D WindowManager: isOnHomeScreen mLastTopComponent: null, componentName: ComponentInfo{com.github.damontecres.wholphin/com.github.damontecres.wholphin.MainActivity}
11-02 21:47:25.199  3690  3798 D WindowManager: isOnHomeScreen mLastTopComponent: null, componentName: ComponentInfo{com.github.damontecres.wholphin/com.github.damontecres.wholphin.MainActivity}
11-02 21:47:25.335  3690  3798 D WindowManager: isOnHomeScreen mLastTopComponent: null, componentName: ComponentInfo{com.github.damontecres.wholphin/com.github.damontecres.wholphin.MainActivity}
11-02 21:47:25.442  3690  3798 D WindowManager: isOnHomeScreen mLastTopComponent: null, componentName: ComponentInfo{com.github.damontecres.wholphin/com.github.damontecres.wholphin.MainActivity}
11-02 21:47:25.549  3690  3798 D WindowManager: isOnHomeScreen mLastTopComponent: null, componentName: ComponentInfo{com.github.damontecres.wholphin/com.github.damontecres.wholphin.MainActivity}
11-02 21:47:25.667  3690  3798 D WindowManager: isOnHomeScreen mLastTopComponent: null, componentName: ComponentInfo{com.github.damontecres.wholphin/com.github.damontecres.wholphin.MainActivity}
11-02 21:47:25.752  3690  3798 D WindowManager: isOnHomeScreen mLastTopComponent: null, componentName: ComponentInfo{com.github.damontecres.wholphin/com.github.damontecres.wholphin.MainActivity}
11-02 21:47:25.867  3690  3798 D WindowManager: isOnHomeScreen mLastTopComponent: null, componentName: ComponentInfo{com.github.damontecres.wholphin/com.github.damontecres.wholphin.MainActivity}
11-02 21:47:26.020  3690  3798 D WindowManager: isOnHomeScreen mLastTopComponent: null, componentName: ComponentInfo{com.github.damontecres.wholphin/com.github.damontecres.wholphin.MainActivity}
11-02 21:47:29.242  3690  3798 D WindowManager: isOnHomeScreen mLastTopComponent: null, componentName: ComponentInfo{com.github.damontecres.wholphin/com.github.damontecres.wholphin.MainActivity}
11-02 21:47:29.389  3690  3798 D WindowManager: isOnHomeScreen mLastTopComponent: null, componentName: ComponentInfo{com.github.damontecres.wholphin/com.github.damontecres.wholphin.MainActivity}
11-02 21:47:32.164  3690  3798 D WindowManager: isOnHomeScreen mLastTopComponent: null, componentName: ComponentInfo{com.github.damontecres.wholphin/com.github.damontecres.wholphin.MainActivity}
11-02 21:47:32.317  3690  3798 D WindowManager: isOnHomeScreen mLastTopComponent: null, componentName: ComponentInfo{com.github.damontecres.wholphin/com.github.damontecres.wholphin.MainActivity}
11-02 21:47:32.468  3690  3798 D WindowManager: isOnHomeScreen mLastTopComponent: null, componentName: ComponentInfo{com.github.damontecres.wholphin/com.github.damontecres.wholphin.MainActivity}
11-02 21:47:32.591  3690  3798 D WindowManager: isOnHomeScreen mLastTopComponent: null, componentName: ComponentInfo{com.github.damontecres.wholphin/com.github.damontecres.wholphin.MainActivity}
11-02 21:47:32.682  3690  3798 D WindowManager: isOnHomeScreen mLastTopComponent: null, componentName: ComponentInfo{com.github.damontecres.wholphin/com.github.damontecres.wholphin.MainActivity}
11-02 21:47:32.793  3690  3798 D WindowManager: isOnHomeScreen mLastTopComponent: null, componentName: ComponentInfo{com.github.damontecres.wholphin/com.github.damontecres.wholphin.MainActivity}
11-02 21:47:32.878  3690  3798 D WindowManager: isOnHomeScreen mLastTopComponent: null, componentName: ComponentInfo{com.github.damontecres.wholphin/com.github.damontecres.wholphin.MainActivity}
11-02 21:47:33.042  3690  3798 D WindowManager: isOnHomeScreen mLastTopComponent: null, componentName: ComponentInfo{com.github.damontecres.wholphin/com.github.damontecres.wholphin.MainActivity}
11-02 21:47:35.765 28486 28517 I OkHttpClient: POST http://192.168.0.209:8096/Users/AuthenticateByName
11-02 21:47:35.966 28486 28511 I OkHttpClient: GET http://192.168.0.209:8096/Users/Me
11-02 21:47:35.972 28486 28511 I OkHttpClient: GET http://192.168.0.209:8096/System/Info
11-02 21:47:35.979 28486 28511 E RawResponse: Deserialization failed
11-02 21:47:35.979 28486 28511 E RawResponse: kotlinx.serialization.MissingFieldException: Field 'overview' is required for type with serial name 'org.jellyfin.sdk.model.api.PackageInfo', but it was missing at path: $.CompletedInstallations[1].PackageInfo
11-02 21:47:35.979 28486 28511 E RawResponse: 	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:95)
11-02 21:47:35.979 28486 28511 E RawResponse: 	at kotlinx.serialization.encoding.AbstractDecoder.decodeSerializableValue(AbstractDecoder.kt:43)
11-02 21:47:35.979 28486 28511 E RawResponse: 	at kotlinx.serialization.encoding.AbstractDecoder.decodeNullableSerializableElement(AbstractDecoder.kt:78)
11-02 21:47:35.979 28486 28511 E RawResponse: 	at org.jellyfin.sdk.model.api.InstallationInfo$$serializer.deserialize(InstallationInfo.kt:20)
11-02 21:47:35.979 28486 28511 E RawResponse: 	at org.jellyfin.sdk.model.api.InstallationInfo$$serializer.deserialize(InstallationInfo.kt:20)
11-02 21:47:35.979 28486 28511 E RawResponse: 	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:69)
11-02 21:47:35.979 28486 28511 E RawResponse: 	at kotlinx.serialization.encoding.AbstractDecoder.decodeSerializableValue(AbstractDecoder.kt:43)
11-02 21:47:35.979 28486 28511 E RawResponse: 	at kotlinx.serialization.encoding.AbstractDecoder.decodeSerializableElement(AbstractDecoder.kt:70)
11-02 21:47:35.979 28486 28511 E RawResponse: 	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableElement(StreamingJsonDecoder.kt:168)
11-02 21:47:35.979 28486 28511 E RawResponse: 	at kotlinx.serialization.encoding.CompositeDecoder.decodeSerializableElement$default(Decoding.kt:539)
11-02 21:47:35.979 28486 28511 E RawResponse: 	at kotlinx.serialization.internal.CollectionLikeSerializer.readElement(CollectionSerializers.kt:80)
11-02 21:47:35.979 28486 28511 E RawResponse: 	at kotlinx.serialization.internal.AbstractCollectionSerializer.readElement$default(CollectionSerializers.kt:51)
11-02 21:47:35.979 28486 28511 E RawResponse: 	at kotlinx.serialization.internal.AbstractCollectionSerializer.merge(CollectionSerializers.kt:36)
11-02 21:47:35.979 28486 28511 E RawResponse: 	at kotlinx.serialization.internal.AbstractCollectionSerializer.deserialize(CollectionSerializers.kt:43)
11-02 21:47:35.979 28486 28511 E RawResponse: 	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:69)
11-02 21:47:35.979 28486 28511 E RawResponse: 	at kotlinx.serialization.encoding.AbstractDecoder.decodeSerializableValue(AbstractDecoder.kt:43)
11-02 21:47:35.979 28486 28511 E RawResponse: 	at kotlinx.serialization.encoding.AbstractDecoder.decodeNullableSerializableElement(AbstractDecoder.kt:78)
11-02 21:47:35.979 28486 28511 E RawResponse: 	at org.jellyfin.sdk.model.api.SystemInfo$$serializer.deserialize(SystemInfo.kt:19)
11-02 21:47:35.979 28486 28511 E RawResponse: 	at org.jellyfin.sdk.model.api.SystemInfo$$serializer.deserialize(SystemInfo.kt:19)
11-02 21:47:35.979 28486 28511 E RawResponse: 	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:69)
11-02 21:47:35.979 28486 28511 E RawResponse: 	at kotlinx.serialization.json.Json.decodeFromString(Json.kt:149)
11-02 21:47:35.979 28486 28511 E RawResponse: 	at org.jellyfin.sdk.api.operations.SystemApi.getSystemInfo(SystemApi.kt:260)
11-02 21:47:35.979 28486 28511 E RawResponse: 	at org.jellyfin.sdk.api.operations.SystemApi$getSystemInfo$1.invokeSuspend(Unknown Source:14)
11-02 21:47:35.979 28486 28511 E RawResponse: 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:34)
11-02 21:47:35.979 28486 28511 E RawResponse: 	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:100)
11-02 21:47:35.979 28486 28511 E RawResponse: 	at kotlinx.coroutines.internal.LimitedDispatcher$Worker.run(LimitedDispatcher.kt:124)
11-02 21:47:35.979 28486 28511 E RawResponse: 	at kotlinx.coroutines.scheduling.TaskImpl.run(Tasks.kt:89)
11-02 21:47:35.979 28486 28511 E RawResponse: 	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:586)
11-02 21:47:35.979 28486 28511 E RawResponse: 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:820)
11-02 21:47:35.979 28486 28511 E RawResponse: 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:717)
11-02 21:47:35.979 28486 28511 E RawResponse: 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:704)
11-02 21:47:35.979 28486 28511 E RawResponse: Caused by: kotlinx.serialization.MissingFieldException: Field 'overview' is required for type with serial name 'org.jellyfin.sdk.model.api.PackageInfo', but it was missing
11-02 21:47:35.979 28486 28511 E RawResponse: 	at kotlinx.serialization.internal.PluginExceptionsKt.throwMissingFieldException(PluginExceptions.kt:20)
11-02 21:47:35.979 28486 28511 E RawResponse: 	at org.jellyfin.sdk.model.api.PackageInfo.<init>(PackageInfo.kt:21)
11-02 21:47:35.979 28486 28511 E RawResponse: 	at org.jellyfin.sdk.model.api.PackageInfo$$serializer.deserialize(PackageInfo.kt:21)
11-02 21:47:35.979 28486 28511 E RawResponse: 	at org.jellyfin.sdk.model.api.PackageInfo$$serializer.deserialize(PackageInfo.kt:21)
11-02 21:47:35.979 28486 28511 E RawResponse: 	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:69)
11-02 21:47:35.979 28486 28511 E RawResponse: 	... 30 more
11-02 21:47:36.503 28486 28511 I OkHttpClient: POST http://192.168.0.209:8096/Users/AuthenticateByName
11-02 21:47:36.701 28486 28511 I OkHttpClient: GET http://192.168.0.209:8096/Users/Me
11-02 21:47:36.707 28486 28511 I OkHttpClient: GET http://192.168.0.209:8096/System/Info
11-02 21:47:36.712 28486 28511 E RawResponse: Deserialization failed
11-02 21:47:36.712 28486 28511 E RawResponse: kotlinx.serialization.MissingFieldException: Field 'overview' is required for type with serial name 'org.jellyfin.sdk.model.api.PackageInfo', but it was missing at path: $.CompletedInstallations[1].PackageInfo
11-02 21:47:36.712 28486 28511 E RawResponse: 	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:95)
11-02 21:47:36.712 28486 28511 E RawResponse: 	at kotlinx.serialization.encoding.AbstractDecoder.decodeSerializableValue(AbstractDecoder.kt:43)
11-02 21:47:36.712 28486 28511 E RawResponse: 	at kotlinx.serialization.encoding.AbstractDecoder.decodeNullableSerializableElement(AbstractDecoder.kt:78)
11-02 21:47:36.712 28486 28511 E RawResponse: 	at org.jellyfin.sdk.model.api.InstallationInfo$$serializer.deserialize(InstallationInfo.kt:20)
11-02 21:47:36.712 28486 28511 E RawResponse: 	at org.jellyfin.sdk.model.api.InstallationInfo$$serializer.deserialize(InstallationInfo.kt:20)
11-02 21:47:36.712 28486 28511 E RawResponse: 	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:69)
11-02 21:47:36.712 28486 28511 E RawResponse: 	at kotlinx.serialization.encoding.AbstractDecoder.decodeSerializableValue(AbstractDecoder.kt:43)
11-02 21:47:36.712 28486 28511 E RawResponse: 	at kotlinx.serialization.encoding.AbstractDecoder.decodeSerializableElement(AbstractDecoder.kt:70)
11-02 21:47:36.712 28486 28511 E RawResponse: 	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableElement(StreamingJsonDecoder.kt:168)
11-02 21:47:36.712 28486 28511 E RawResponse: 	at kotlinx.serialization.encoding.CompositeDecoder.decodeSerializableElement$default(Decoding.kt:539)
11-02 21:47:36.712 28486 28511 E RawResponse: 	at kotlinx.serialization.internal.CollectionLikeSerializer.readElement(CollectionSerializers.kt:80)
11-02 21:47:36.712 28486 28511 E RawResponse: 	at kotlinx.serialization.internal.AbstractCollectionSerializer.readElement$default(CollectionSerializers.kt:51)
11-02 21:47:36.712 28486 28511 E RawResponse: 	at kotlinx.serialization.internal.AbstractCollectionSerializer.merge(CollectionSerializers.kt:36)
11-02 21:47:36.712 28486 28511 E RawResponse: 	at kotlinx.serialization.internal.AbstractCollectionSerializer.deserialize(CollectionSerializers.kt:43)
11-02 21:47:36.712 28486 28511 E RawResponse: 	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:69)
11-02 21:47:36.712 28486 28511 E RawResponse: 	at kotlinx.serialization.encoding.AbstractDecoder.decodeSerializableValue(AbstractDecoder.kt:43)
11-02 21:47:36.712 28486 28511 E RawResponse: 	at kotlinx.serialization.encoding.AbstractDecoder.decodeNullableSerializableElement(AbstractDecoder.kt:78)
11-02 21:47:36.712 28486 28511 E RawResponse: 	at org.jellyfin.sdk.model.api.SystemInfo$$serializer.deserialize(SystemInfo.kt:19)
11-02 21:47:36.712 28486 28511 E RawResponse: 	at org.jellyfin.sdk.model.api.SystemInfo$$serializer.deserialize(SystemInfo.kt:19)
11-02 21:47:36.712 28486 28511 E RawResponse: 	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:69)
11-02 21:47:36.712 28486 28511 E RawResponse: 	at kotlinx.serialization.json.Json.decodeFromString(Json.kt:149)
11-02 21:47:36.712 28486 28511 E RawResponse: 	at org.jellyfin.sdk.api.operations.SystemApi.getSystemInfo(SystemApi.kt:260)
11-02 21:47:36.712 28486 28511 E RawResponse: 	at org.jellyfin.sdk.api.operations.SystemApi$getSystemInfo$1.invokeSuspend(Unknown Source:14)
11-02 21:47:36.712 28486 28511 E RawResponse: 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:34)
11-02 21:47:36.712 28486 28511 E RawResponse: 	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:100)
11-02 21:47:36.712 28486 28511 E RawResponse: 	at kotlinx.coroutines.internal.LimitedDispatcher$Worker.run(LimitedDispatcher.kt:124)
11-02 21:47:36.712 28486 28511 E RawResponse: 	at kotlinx.coroutines.scheduling.TaskImpl.run(Tasks.kt:89)
11-02 21:47:36.712 28486 28511 E RawResponse: 	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:586)
11-02 21:47:36.712 28486 28511 E RawResponse: 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:820)
11-02 21:47:36.712 28486 28511 E RawResponse: 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:717)
11-02 21:47:36.712 28486 28511 E RawResponse: 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:704)
11-02 21:47:36.712 28486 28511 E RawResponse: Caused by: kotlinx.serialization.MissingFieldException: Field 'overview' is required for type with serial name 'org.jellyfin.sdk.model.api.PackageInfo', but it was missing
11-02 21:47:36.712 28486 28511 E RawResponse: 	at kotlinx.serialization.internal.PluginExceptionsKt.throwMissingFieldException(PluginExceptions.kt:20)
11-02 21:47:36.712 28486 28511 E RawResponse: 	at org.jellyfin.sdk.mod
```

--- a/app/src/main/java/com/github/damontecres/wholphin/data/ServerRepository.kt
+++ b/app/src/main/java/com/github/damontecres/wholphin/data/ServerRepository.kt
@@ -84,13 +84,15 @@ class ServerRepository
             }
             Timber.v("Changing user to ${user.name} on ${server.url}")
             apiClient.update(baseUrl = server.url, accessToken = user.accessToken)
-            val userDto =
-                apiClient.userApi
-                    .getCurrentUser()
-                    .content
-            val sysInfo by apiClient.systemApi.getSystemInfo()
-
-            val updatedServer = server.copy(name = sysInfo.serverName)
+            val userDto by apiClient.userApi.getCurrentUser()
+            val updatedServer =
+                try {
+                    val sysInfo by apiClient.systemApi.getPublicSystemInfo()
+                    server.copy(name = sysInfo.serverName)
+                } catch (ex: Exception) {
+                    Timber.w(ex, "Exception fetching public system info")
+                    server
+                }
             var updatedUser =
                 user.copy(
                     id = userDto.id,
@@ -171,8 +173,14 @@ class ServerRepository
                         }
                     if (user != null) {
                         changeUser(server, user)
+                    } else {
+                        throw IllegalArgumentException("Authentication result's user was null")
                     }
+                } else {
+                    throw IllegalArgumentException("Authentication result's serverId not valid: ${authenticationResult.serverId}")
                 }
+            } else {
+                throw IllegalArgumentException("Authentication result's access token was null")
             }
         }
 

